api/v1alpha1/provisioning_types.go

```diff
@@ -48,6 +48,22 @@ const (
 	BootIsoSourceHttp  BootIsoSource = "http"
 )
 
+// PreprovisioningOSDownloadURLs allows specification of a set of
+// OS URLs that can be used by metal3 to pre-provision workers
+type PreprovisioningOSDownloadURLs struct {
+	// Image URL to be used for live ISO deployments
+	LiveISO string `json:"liveISO,omitempty"`
+
+	// Kernel Image URL to be used for PXE deployments
+	Kernel string `json:"kernel,omitempty"`
+
+	// InitRD Image URL to be used for PXE deployments
+	InitRD string `json:"initRD,omitempty"`
```

Comment:
This should be called InitramFS to match the exact file name. Initrd is an alternate name but here we're explicit about names and the other three keys here match the file names except LiveISO, which should just be ISO, IMHO.

Reply:
Updated based on the above comment.